runtime/lua/vim/lsp/util.lua

```diff
@@ -1784,7 +1784,7 @@ local function make_position_param()
   if not line then
     return { line = 0; character = 0; }
   end
-  col = str_utfindex(line, col)
+  _, col = str_utfindex(line, col)
```

Comment:
this is the correct default, but ideally this should use the configured position encoding by the client (which will be plain ol' bytes for clangd), which is gonna need a parameter to this function.

Reply:
working on that

Reply:
I think this is the only thing that should change in this PR.